### DISCUSSION TITLE
Need help identifying controller

### DISCUSSION THREAD

**Discussion by jakeiscrazy** [jgyates/genmon#1090] [Questions]
I have a Generac QT005219-0 generator that I would like to integrate with Genmon. However I cannot figure out what controller I have on it or if it may be compatible. Any help would be greatly appreciated!

![image](https://github.com/jgyates/genmon/assets/34757421/582bbdcf-aec4-4f02-9735-94663d143e60)

![image](https://github.com/jgyates/genmon/assets/34757421/14320199-c515-4aae-b80c-25d3ba858c97)

**Comment by jgyates** [comment]
Unfortunately that controller looks like an earlier model that does not support serial modbus so genmon will not support this controller. Generac started putting serial ports that support modbus on the controllers around 2008. Your controller looks like an R-200.

https://gensetservices.com/wp-content/themes/gensetservices/PartsPDFs/0G1341.pdf

**Reply by jakeiscrazy** [reply]
Bummer! Yeah I think this is was built in 05. Well I may upgrade the controller, or maybe just build something to read the lead status lights
